docs/sources/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -93,7 +91,7 @@ The store-gateway stores each block’s index-header on the local disk and loads
 The store-gateway keeps a file descriptor open for each index-header loaded at a given time.
 The total number of file descriptors used to load index-headers linearly increases with the number of blocks owned by the store-gateway instance.
 
-We recommend configuring the system's `file-max` ulimit at least to `65536` to avoid reaching the maximum number of open file descriptors.
+Configure the system's `file-max` ulimit at least to `65536` to avoid reaching the maximum number of open file descriptors.
```

Comment:
```suggestion
To avoid reaching the maximum number of open file descriptors, configure the system's `file-max` ulimit to at least `65536`.
```

Reply:
Rationale (I think, as a native speaker who sometimes does not know why) for changing the word order: "at least" is more tightly coupled / collocated with the value `65536` than it is to "ulimit". If any grammar nerds want to pipe up, I’d be curious why I changed this. :)